apps/librepcb-cli/CMakeLists.txt

```diff
@@ -4,59 +4,64 @@ set(CMAKE_AUTOUIC OFF)
 set(CMAKE_AUTORCC OFF)
 
 # Executable
-add_executable(librepcb_cli
-    # When building on macOS, create a bundle
```

Comment:
This comment got removed. But I assume it should be self-explanatory.

Reply:
Re-added the comment (it's now on the same line as `MACOSX_BUNDLE` and the target name, but I'd say that's OK).